curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64d0a6174ffbae02090bad27.md

```diff
@@ -0,0 +1,465 @@
+---
+id: 64d0a6174ffbae02090bad27
+title: Step 73
+challengeType: 0
+dashedName: step-73
+---
+
+# --description--
+
+To detect a full house, you need to create a function named `detectFullHouse` with an argument of `arr`. Then, create a variable named `counts` and set it to an empty object.
+
+# --hints--
+
+You need to create a function named `detectFullHouse` with an argument of `arr`.
+
+```js
+assert.match(code, /function\s*detectFullHouse\s*\(\s*arr\s*\)\s*{/);
```

Comment:
Test should be added for the arrow function syntax as it is also used in the seed code.

The regex(`/const\s*detectFullHouse\s*=\s*\(arr\)\s*=>\s*{\s*(let|var|const)\s*counts\s*=\s*{\s*}\s*;?\s*}/`) works for:

```
const detectFullHouse = (arr) => {
  const counts = {};
  }
```